Assignment/SampleData.cs

```diff
@@ -11,32 +11,32 @@ public enum CsvColumn { Id, FirstName, LastName, Email, StreetAddress, City, Sta
         private const string _CSV_FILENAME = "People.csv";

```

Comment:
I think private variables are supposed to be in pascal case, eg 
```suggestion
         private const string _CsvFileName = "People.csv";
```